data/controller_data.py

```diff
@@ -201,4 +207,10 @@
      'image_url': 'http://www.jese.co.uk/product/image/rdmtri.gif',
      'tags': ['tool', 'RDM Responder Tests', 'USB']},
   ],
+  0x736c: [
+    {'name': 'NEO',
+     'link': 'https://www.strandlighting.com/global/products/neo-console-and-software',
+     'image_url': 'https://www.strandlighting.com/b-dam/strand-lighting/products/neo-console-and-software/NEO-Console-452x410.jpg'},
+    'tags': ['console']},
```

Comment:
Can you fix the spacing of this please (indent by one).

Reply:
Done.